.github/workflows/pr-checks.yml

```
@@ -0,0 +1,260 @@
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: 'npm'
+          
+      - name: Install dependencies
+        run: npm ci
+        
+      - name: Build PR bundle
+        run: npm run build
+        
+      - name: Analyze bundle size
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:4:28: Double quote to prevent globbing and word splitting

(shellcheck)